qemu/tests/qemu_guest_agent.py

```diff
@@ -1309,13 +1309,14 @@ def setup(self, test, params, env):
         BaseVirtTest.setup(self, test, params, env)
         if self.start_vm == "yes":
             session = self._get_session(params, self.vm)
+            qemu_ga_pkg_path = self.get_qga_pkg_path(test, session, params, self.vm)
+            self.gagent_install_cmd = params.get("gagent_install_cmd") % qemu_ga_pkg_path
+            self.gagent_uninstall_cmd = params.get("gagent_uninstall_cmd") % qemu_ga_pkg_path
```

Comment:
please explain issue in commit message clearly,
In this case you need a additional check for `Windows` vm as linux vm install/uninstall cmds does not take any additional arguments.

it is better to handle initialization in below part where assignment already happens, may be additional vm login needs to be handled for windows vm.
 

```
class QemuGuestAgentTest(BaseVirtTest):

    def __init__(self, test, params, env):
        BaseVirtTest.__init__(self, test, params, env)

        self._open_session_list = []
        self.gagent = None
        self.vm = None
        self.gagent_install_cmd = params.get("gagent_install_cmd")
        self.gagent_uninstall_cmd = params.get("gagent_uninstall_cmd")

Reply:
sathnaga  could you help review again?

Reply:
sathnaga@linux.vnet.ibm.com could you help review again?

Reply:
@sathnaga could you help review again?